docs/governance/03-voting_process/01-linking_mainchain_address.md

```diff
@@ -4,4 +4,156 @@ title: Linking Mainchain and EON $ZEN Addresses
 
 Because Snapshot is not compatible with Horizen’s mainchain, it has been integrated into EON, which is Ethereum-compatible. This means that $ZEN tokenholders will need to “link” their mainchain $ZEN address with an EON address in order to capture their full voting power for proposals in Snapshot. 
 
-Instructions Coming Soon!
\ No newline at end of file
+## Overview
+
+To enable voting power to all accounts within the Horizen ecosystem, the **Governance Linking** app provides a mechanism to link multiple mainchain addresses to one  EON address. Upon linking, a user’s voting power for the Horizen DAO will be the sum total of their ZEN balance on EON, plus the ZEN balance of all of their linked mainchain addresses. This makes it easier for users to vote with the totality of their ZEN without having to move assets. 
+
+### Configuring a Network
+
+Before linking an EON wallet, make sure that the appropriate network is available. If not, add the network to the EON wallet and set it as the primary network. 
+
+See [Connect your Wallet](https://docs.horizen.io/horizen_eon/connect/connect_your_wallet) in the EON documentation.
+
+
+## Linking Addresses 
+
+Perform the following steps to link one EON address to one or more mainchain addresses.
+
+**Note:** MetaMask is used for providing the EON address.
+
+
+
+1. Open the **[Horizen Governance ](https://development.evm-backward-transfer-ui-e76.pages.dev/governance/dao)**page. 
```

Comment:
@Ramone-HL currently this links to a dev/preview link. Also, its [production page](https://eon.horizen.io/governance/dao) counterpart  is hidden/not accessible through the sidebar.

@yurikoinaba are we making the [Governance page](https://eon.horizen.io/governance/dao) public with this change? I also noticed that most of the buttons like the "Mainchain Link Guide", "Snapshot", and "Discourse" don't do anything when clicked and will require an update.

Reply:
The production link will be: https://eon.horizen.io/governance/dao and is not available.

Reply:
For Discourse the link is:  http://forum.horizen.io/